_layouts/website/page.html

```diff
@@ -53,7 +53,7 @@ <h3 class="panel-title">{{ "FAQ_RELATED_ARTICLES"|t }}</h3>
                     {% for related in page.related %}
                         {% set relatedArticle = getArticleByPath(related) %}
                         {% if relatedArticle %}
-                        <a href="{{ relatedArticle.path|resolveFile }}" class="list-group-item">{{ relatedArticle.title }}</a>
+                        <a href="{{ relatedArticle.path|resolveFile }}{{ relatedArticle.anchor }}" class="list-group-item">{{ relatedArticle.title }}</a>
```

Comment:
This will find the first article with matching path, and use this article's anchor. But that's not really what we would like here. If we have several articles pointing to the same file, we should expect the `related` property to have the anchor included, so that `related` refer without ambiguity to a particular article.

So we should do something like `getArticleByPathWithAnchor(related)`... but this is not in the API. So I would not implement it until GitBook v4 is finished.

Do you have a use-case for this?